src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -56,4 +51,11 @@ await dispatcher.Dispatch(new TransportOperations(operation), transportTransacti
 
     static ILog log = LogManager.GetLogger<NServiceBusMetricReport>();
     TransportTransaction transportTransaction = new TransportTransaction();
-}
+
+    static Dictionary<string, string> headers = new Dictionary<string, string>
+    {
+        { Headers.EnclosedMessageTypes, "NServiceBus.Metrics.MetricReport" }, // without assembly name to allow ducktyping
+        { Headers.ContentType, ContentTypes.Json },
+        { Headers.MessageIntent, MessageIntentEnum.Send.ToString() }
```

Comment:
Not sure about `MessageIntent`. The receiving endpoint does not strictly need it and I am not convinced the semantics are the same as for send. We put it into a single queue but there is no command semantics associated with the message IMO.